.github/workflows/workflow.yaml

```diff
@@ -104,30 +104,35 @@ jobs:
           if [[ $exists == null ]]
           then
             make docker-login
-            make docker-build IMG=gatekeeper-e2e:latest
-            make docker-push-dev IMG=gatekeeper-e2e:latest DEV_TAG=${GITHUB_SHA::7}
+            make docker-buildx-dev DEV_TAG=${GITHUB_SHA::7}
           fi
         env:
           DOCKER_USER: ${{ secrets.DOCKER_USER }}
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
 
-      - uses: "marvinpinto/action-automatic-releases@v1.0.0"
+      - uses: 'marvinpinto/action-automatic-releases@v1.0.0'
         with:
-          repo_token: "${{ secrets.GITHUB_TOKEN }}"
+          repo_token: '${{ secrets.GITHUB_TOKEN }}'
           prerelease: true
           automatic_release_tag: latest
-          title: "Development Build"
+          title: 'Development Build'
 
   tagged-release:
-    name: "Tagged Release"
-    runs-on: "ubuntu-latest"
+    name: 'Tagged Release'
+    runs-on: 'ubuntu-latest'
     if: startsWith(github.ref, 'refs/tags/v') && github.repository == 'open-policy-agent/gatekeeper'
     needs: [lint, build_test, helm_build_test]
     timeout-minutes: 30
     steps:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
 
+      - name: Set up Docker Buildx
```

Comment:
why is this only in tagged-release? doesn't make `docker-buildx-release` already handle this for us since it does create the builder and it'll get overridden since it has `--use`

Reply:
The Makefile ref is so for reproducibility when running locally. We do need to add this to the other job though.